### DISCUSSION TITLE
my wire:click (livewire 3) does not work, when I use the boostrap data table

### DISCUSSION THREAD

**Discussion by divinmik** [livewire/livewire#7149] [1. Help]
My wire:click button does not work when using the DataTable (https://datatables.net/download/)

One can help me

**Comment by lrljoe** [comment]
Not overly familiar with that package, but I imagine it's because you haven't initialised Livewire on the page.

There are a few native livewire tables packages out there that may prove useful.

Otherwise, precise error messages from your browser console will be helpful!

**Reply by divinmik** [reply]
Thank you for your support,

Excuse me for my englais, 

I have initialized the livewire packages (@livewireStyle and @livewireScript).

When I remove the tag responsible for the DataTable, my wire:click button functions correctly.

I’ll guide my searcher to the table packages.

![livewire import](https://github.com/livewire/livewire/assets/39834399/b61d0944-6c83-4d6d-acc3-63e3f0000a44)
![table edit](https://github.com/livewire/livewire/assets/39834399/ca9fece7-4e1d-49ef-a268-3cb581051698)

**Reply by shubdragon** [reply]
check console

**Comment by KhoirulAnam78** [comment]
I had the same problem, my wire:click inside datatables not responding, there is a solution for this ?
